src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -219,6 +219,21 @@
                     "parameters": []
                 }
             ]
+        },
+        {
+            "path": "/v1/debug/restart_service",
+            "operations": [
+                {
+                    "method": "PUT",
+                    "summary": "Restart a redpanda service",
+                    "type": "void",
+                    "nickname": "restart_service",
+                    "produces": [
+                        "application/json"
+                    ],
+                    "parameters": []
```

Comment:
same comment as @michael-redpanda are we not copying over the parameter from the previous endpoint definition for some reason?